server/uploadHandler/src/service/S3Client.ts

```diff
@@ -52,11 +52,11 @@ export const getUploadedFilesInJob = async (jobId: string): Promise<string[]> =>
 };
 
 const getContentType = (fileExtension: string) => {
-	if (videoExtension.includes(fileExtension.toLocaleLowerCase())) {
+	if (videoExtension.includes(fileExtension.toLowerCase())) {
 		return "video/mp4";
 	}
-	if (imageExtension.includes(fileExtension.toLocaleLowerCase())) {
-		return `image/${fileExtension.toLocaleLowerCase}`;
+	if (imageExtension.includes(fileExtension.toLowerCase())) {
+		return `image/${fileExtension.toLowerCase()}`;
```

Comment:
This is causing my issue, all jpg extensions should map to the image/jpeg MIME type. See https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types
This is causing file uploads to fail if the extension isn't exactly ".jpeg" (".jpg" doesn't work)

Reply:
You'll also need to add support for PNG's (`image/png`)